pkg/util/sysctl/sysctl.go

```diff
@@ -24,25 +24,41 @@ import (
 )
 
 const (
-	sysctlBase         = "/proc/sys"
-	VmOvercommitMemory = "vm/overcommit_memory"
-	VmPanicOnOOM       = "vm/panic_on_oom"
-	KernelPanic        = "kernel/panic"
-	KernelPanicOnOops  = "kernel/panic_on_oops"
-	RootMaxKeys        = "kernel/keys/root_maxkeys"
-	RootMaxBytes       = "kernel/keys/root_maxbytes"
+	sysctlBase = "/proc/sys"
+	// VMOvercommitMemory defines the memory over-commit policy used by kernel.
```

Comment:
Nit: Imho, For many of these, the comment makes it appear as if the constant defines the actual value of the variable, as opposed to the name of the variable.

For example, for `VMOvercommitMemory` could we say `VMOvercommitMemory refers to the sysctl variable responsible for defining the memory over-commit policy used by kernel`.